docs/middleware/ics29-fee/fee-distribution.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 4
+-->
+
+# Fee Distribution
```

Comment:
nit: trying to follow [Google documentation style guide](https://developers.google.com/style/headings) (_Use sentence case for headings and titles._). I promise I will also add to the repo a documentation style guide to document all these little nits.

```suggestion
# Fee distribution
```